bin/dart-dbus.dart

```diff
@@ -644,15 +646,15 @@ String generateRemoteSignalSubscription(
 /// Converts a D-Bus path to a Dart class name. e.g. 'org.freedesktop.Notifications' -> 'OrgFreedesktopNotifications'.
 String pathToClassName(String path) {
   var className = '';
-  for (var element in path.split('/')) {
+  for (var element in path.split(RegExp('[/\.]'))) {
```

Comment:
This made me notice the function is wrongly named - it was actually `interfaceToClassName` and now it's `interfaceOrPathToClassName`. But I'll follow that up in a following commit.